descriptors/ModuleDescriptor-template.json

```diff
@@ -141,7 +141,7 @@
     },
     {
       "id": "resource-ids-streaming",
-      "version": "0.3",
+      "version": "0.4",
```

Comment:
I don't think we should change the interface version. Because making it async is not a new feature, it is supposed to work this way. Is there any other reason to bump version?

Reply:
I thought the fix is a reason.

Reply:
No. We change the version only when we change expectations of API behaviour.